joss_paper.bib

```diff
@@ -14,16 +14,15 @@ @article{lindsay2022moose
   volume = {20},
    pages = {101202},
     issn = {2352-7110},
-     doi = {https://doi.org/10.1016/j.softx.2022.101202},
-     url = {https://www.sciencedirect.com/science/article/pii/S2352711022001200},
+     doi = {10.1016/j.softx.2022.101202},
 keywords = {Multiphysics, Object-oriented, Finite-element, Framework},
 }
 
 @techreport{relap7theory,
 title = {{RELAP-7} Theory Manual},
 author = {R. A. Berry and L. Zou and H. Zhao and H. Zhang and J. W. Peterson and R. C. Martineau and S. Y. Kadioglu and D. Andrs},
 institution = {Idaho National Laboratory},
-number = {INL/EXT-14-31366},
+howpublished = {INL/EXT-14-31366},
```

Comment:
I think `number` is the appropriate field - why change?

Reply:
ah, I thought that our citation formatting didn't recognize `number` for `techreport` type, but at second glance I can see that it did show up property. More importantly, we should add the DOI 10.2172/1262488